package  exception.RuntimeException;

public class RunTimeExceptionTest01 {
    public static void main(String[] args) {

    }
}
